docs/design/resources.md

```diff
@@ -94,3 +96,25 @@ following:
 5. The cluster resource eventually is able to communicate with the
    cluster and comes up.
 
+### Scale-up
+
+1. A user edits the `EtcdCluster` resource to increase the size by
+   one.
+2. Using the logic above, the cluster operator notes that the cluster
+   is undersized by one and adds a new peer to the etcd cluster.
```

Comment:
I've (loosely) tried to use the term 'member' to refer to an entry in the etcd membership API, and 'peer' to refer to an EtcdPeer/pod. Not sure if that terminology makes sense to you, and if you want to use it too 😄

Reply:
+1 lets distinguish these terms, not clear here if `new peer` means new EtcdPeer resource or new etcd cluster member

Reply:
Clarified.